doc/integrator/upgrade_application.rst

```diff
@@ -29,17 +29,30 @@ Where ``<makefile>`` is your user make file (``<user>.mk``).
 Easy upgrading an application
 -----------------------------
 
-Verify that you have in your ``project.yaml.mako`` file the following ``template_vars``: ``package``, ``srid``, ``extent``, ``mobile_application_title``, ``apache_vhost``, for example:
+Verify that you have in your ``project.yaml.mako`` file like the following:
```

Comment:
Verify that your ``project.yaml.mako`` file is like the following:
ou bien
Verify that you have in your ``project.yaml.mako`` file the following content: